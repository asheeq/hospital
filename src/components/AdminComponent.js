import React from 'react'
import { Link } from 'react-router-dom'

function AdminComponent() {
    return (
        <>
            <div className="container">
                <h1>Admin Panel</h1>
                <div className="container">
                    <Link to="/admin/appointments"><button class="button button6"><span>Appointments</span></button></Link>
                    <Link to="/admin/patients"><button class="button button6"><span>Patients</span></button></Link>
                    <Link to="/admin/doctors"><button class="button button6"><span>Doctors</span></button></Link>
                </div>
            </div>
        </>
    )  
}

export default AdminComponent