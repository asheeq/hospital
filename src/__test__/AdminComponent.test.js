import React from "react";
import { shallow } from "enzyme";
import AdminComponent from '../components/AdminComponent.js'
import { Link } from 'react-router-dom'
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router'

it("Renders without crashing", () => {
    shallow(<AdminComponent />);
});

test('Link matches snapshot', () => {
  const component = renderer.create(
    <StaticRouter>
      <Link to="/admin/patients" />
      <Link to="/admin/doctors" />
      <Link to="/admin/appointments" />
    </StaticRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});