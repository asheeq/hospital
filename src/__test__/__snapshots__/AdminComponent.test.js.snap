// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link matches snapshot 1`] = `
Array [
  <a
    href="/admin/patients"
    onClick={[Function]}
  />,
  <a
    href="/admin/doctors"
    onClick={[Function]}
  />,
  <a
    href="/admin/appointments"
    onClick={[Function]}
  />,
]
`;
